code/drasil-printers/Language/Drasil/Printing/Import.hs

```diff
@@ -90,76 +92,80 @@ processExpo a
 
 -- | expr translation function from Drasil to layout AST
 expr :: Expr -> PrintingInformation -> P.Expr
-expr (Dbl d)              sm = case sm ^. getSetting of
+expr (Dbl d)                 sm = case sm ^. getSetting of
   Engineering -> P.Row $ digitsProcess (map toInteger $ fst $ floatToDigits 10 d)
      (fst $ processExpo $ snd $ floatToDigits 10 d) 0
      (toInteger $ snd $ processExpo $ snd $ floatToDigits 10 d)
   Scientific  ->  P.Dbl d
-expr (Int i)               _ = P.Int i
-expr (Str s)               _ = P.Str s
-expr (Perc a b)           sm = P.Row [expr (Dbl val) sm, P.MO P.Perc]
+expr (Int i)                  _ = P.Int i
+expr (Str s)                  _ = P.Str s
+expr (Perc a b)              sm = P.Row [expr (Dbl val) sm, P.MO P.Perc]
   where
     val = fromIntegral a / (10 ** fromIntegral (b - 2))
-expr (AssocB And l)       sm = P.Row $ intersperse (P.MO P.And) $ map (expr' sm (precB And)) l
-expr (AssocB Or l)        sm = P.Row $ intersperse (P.MO P.Or ) $ map (expr' sm (precB Or)) l
-expr (AssocA Add l)       sm = P.Row $ intersperse (P.MO P.Add) $ map (expr' sm (precA Add)) l
-expr (AssocA Mul l)       sm = P.Row $ mulExpr l sm
-expr (Deriv Part a b)     sm =
+expr (AssocB And l)          sm = 
```

Comment:
In general, if you can keep these on a single line, that would be nice. The lines can go slightly over 80 characters if needed.

Another way would be to make a helper function for the pattern 
```haskell
P.Row $ intersperse (P.MO x) $ map (expr' sm (precB y)) l
```
used 3 times in the above.

Reply:
Thanks! I made a helper function along these lines to resolve this.